packages/compass-connections-navigation/src/connections-navigation-tree.tsx

```diff
@@ -497,6 +488,8 @@ const ConnectionsNavigationTree: React.FunctionComponent<
     return data.items[index].key;
   }, []);
 
+  const isTestEnv = process.env.NODE_ENV === 'test';
```

Comment:
Now that I see this, is this going to be a problem in compass-web? Are the tests for compass-web working with this change?

Reply:
do they have a different env value?
for e2e tests - the `height` & `width` shouldn't apply. they are only used as default, if there is no "real" size available. the problem with integration tests was autosize requires a container with fixed size. in it's absence it doesn't take all available space, but collapses to nothing. this is what the defaults are preventing.
the only change for e2e tests would be `overscanCount`, which I don't understand yet and need to check.